src/app/views/query-runner/query-input/QueryInput.tsx

```diff
@@ -78,56 +94,87 @@ const QueryInput = (props: IQueryInputProps) => {
     handleOnRunQuery(query);
   };
 
-  const queryInputStackTokens: IStackTokens = {
-    childrenGap: 10
-  };
-
-
   return (
-    <>
-      <Stack horizontal={mobileScreen ? false : true} tokens={queryInputStackTokens} horizontalAlign='space-between'>
-        <Stack.Item styles={!mobileScreen ? queryButtonStyles : {}}>
-          <Dropdown
-            ariaLabel={translateMessage('HTTP request method option')}
-            selectedKey={sampleQuery.selectedVerb}
-            options={httpMethods}
-            styles={verbSelector}
-            errorMessage={showError ? translateMessage('Sign in to use this method') : undefined}
-            onChange={(event, method) => handleOnMethodChange(method)}
-          />
-        </Stack.Item>
-        <Stack.Item >
-          <Dropdown
-            ariaLabel={translateMessage('Microsoft Graph API Version option')}
-            selectedKey={sampleQuery.selectedVersion || GRAPH_API_VERSIONS[0]}
-            options={urlVersions}
-            onChange={(event, method) => handleOnVersionChange(method)}
-          />
-        </Stack.Item>
-        <Stack.Item grow>
-          <AutoComplete
-            contentChanged={contentChanged}
-            runQuery={runQuery}
-          />
-        </Stack.Item>
-        <Stack.Item shrink>
-          <SubmitButton
-            className='run-query-button'
-            text={translateMessage('Run Query')}
-            disabled={showError || !sampleQuery.sampleUrl || !validation.isValid}
-            role='button'
-            handleOnClick={() => runQuery()}
-            submitting={isLoadingData}
-            allowDisabledFocus={true}
-          />
-        </Stack.Item>
-        <Stack.Item shrink styles={!mobileScreen ? queryButtonStyles : {}}>
-
-        </Stack.Item>
-      </Stack>
-    </>
-  )
-}
+    <div
+      className={classes.container}
+      style={{
+        flexDirection: mobileScreen ? 'column' : 'row'
+      }}
+    >
+      <Field
+        validationMessageIcon={showError ? <ErrorCircle12Filled /> : null}
+        validationMessage={
+          showError ? translateMessage('Sign in to use this method') : undefined
+        }
+        validationState={showError ? 'error' : 'none'}
+      >
+        <Dropdown
+          aria-label={translateMessage('HTTP request method option')}
+          placeholder='Select method'
+          value={sampleQuery.selectedVerb}
+          className={classes.smallDropdown}
+          onOptionSelect={(event, data) => {
```

Comment:
You can use the button slot to style the dropdown value color to match the badge colors
```suggestion
          button={{ style: { color: selectedBadgeColor } }}
          onOptionSelect={(event, data) => {
```

then define the selected badge color as
 `const selectedMethod = sampleQuery.selectedVerb;
  const selectedBadgeColor = getStyleFor(selectedMethod);`

Then update the getStyleFor method to use v9 tokens 

```
export function getStyleFor(method: string) {
  method = method?.toUpperCase();
  const styles: Record<string, string> = {
    GET: tokens.colorBrandBackground,
    POST: tokens.colorStatusSuccessForeground1,
    PUT: tokens.colorStatusWarningForeground2,
    PATCH: tokens.colorStatusWarningForeground3,
    DELETE: tokens.colorStatusDangerForeground1
  };

  return styles[method] || tokens.colorNeutralForeground1;
}
```